apps/cms-v2/src/modules/providers/contentful/contentful.router.ts

```diff
@@ -38,7 +38,11 @@ export const contentfulRouter = router({
         space: input.contentfulSpace,
       });
 
-      return client.getEnvironments();
+      return client.getEnvironments().catch((e) => {
+        console.error(e);
```

Comment:
**question**: was this committed intentionally?

Reply:
I will replace with logger